TACC-Cloud/geoapi
# Task/DES-1760 Tile Servers Backend

## Overview: ##
Backend for persisting external tile server data.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1760](https://jira.tacc.utexas.edu/browse/DES-1760)

## Summary of Changes: ##
Added tile server related api handles to `services/features.py` and `routes/project.py` and created a tile server model.
For tests, added tests to `test_feature_service.py` and created `test_tile_server_routes.py`. Also added a `metadata.ini` fixture for the test.

## Testing Steps: ##
1. See that tests runs properly.
2. Try to use it with frontend

## Notes: ##
